index.js

```diff
@@ -37,16 +37,15 @@ function build () {
       }
     }
 
-    emitted.set(code, false)
```

Comment:
Why is this removed?

Reply:
Because we add the code to the Set if it was emitted. 

A Map with true and false values is basically a Set, where true is in the Set and false is not in the Set.